web/tests/Web.Integration.Tests/BenchmarkingWebAppWebAppClient.cs

```diff
@@ -61,6 +65,42 @@ protected override void Configure(IServiceCollection services)
         services.AddSingleton(BudgetForecastApi.Object);
         services.AddSingleton(HttpContextAccessor.Object);
         services.AddSingleton(DataSourceStorage.Object);
+        services.AddSingleton(FeatureManager.Object);
+
+        EnableFeatures();
+    }
+
+    private void EnableFeatures()
+    {
+        var features = new[]
+        {
+            FeatureFlags.CurriculumFinancialPlanning,
+            FeatureFlags.CustomData,
+            FeatureFlags.UserDefinedComparators,
+            FeatureFlags.TrustComparison,
+            FeatureFlags.Trusts,
+            FeatureFlags.LocalAuthorities,
+            FeatureFlags.ForecastRisk
+
+        };
+
+        foreach (var feature in features)
+        {
+            FeatureManager.Setup(fm => fm.IsEnabledAsync(feature))
+                .ReturnsAsync(true);
+        }
+    }
+
+    public BenchmarkingWebAppClient SetupDisableFeatureFlags(string featureFlag)
+    {
+        FeatureManager.Reset();
+
+        EnableFeatures();
```

Comment:
Let's have a chat about this, as it doesn't feel right to enable and then disable a feature.

Reply:
Thanks for this. As discussed amended these methods and relative paths used as discussed in the latest commits